automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/SignInSpec.scala

```diff
@@ -2,23 +2,28 @@ package org.broadinstitute.dsde.firecloud.test.user
 
 import org.broadinstitute.dsde.firecloud.config.{AuthToken, AuthTokens, Config}
 import org.broadinstitute.dsde.firecloud.test.{CleanUp, WebBrowserSpec}
+import org.broadinstitute.dsde.firecloud.fixture.UserFixtures
 import org.scalatest._
+import org.broadinstitute.dsde.firecloud.test.Tags
 
-class SignInSpec extends FreeSpec with WebBrowserSpec with CleanUp {
 
-  implicit val authToken: AuthToken = AuthTokens.harry
+class SignInSpec extends FreeSpec with WebBrowserSpec with UserFixtures with CleanUp {
+
+  // implicit val authToken: AuthToken = AuthTokens.draco
 
   "A user" - {
     "with a registered account" - {
 
-      "should be able to log in and out multiple times as multiple users" in withWebDriver { implicit driver =>
-        var listPageAsUser1 = signIn(Config.Users.harry)
-        listPageAsUser1.signOut()
-        val listPageAsUser2 = signIn(Config.Users.ron)
-        assert(listPageAsUser2.readUserEmail().equals(Config.Users.ron.email))
-        listPageAsUser2.signOut()
-        listPageAsUser1 = signIn(Config.Users.harry)
-        assert(listPageAsUser1.readUserEmail().equals(Config.Users.harry.email))
+      "should be able to log in and out multiple times as multiple users" taggedAs Tags.GooglePassing in withWebDriver { implicit driver =>
```

Comment:
This can be rewritten a bit more fluently with `withSignIn`:
```
      "should be able to log in and out multiple times as multiple users" taggedAs Tags.GooglePassing in withWebDriver { implicit driver =>
        withSignIn(Config.Users.draco) { listPage =>
          listPage.readUserEmail() shouldEqual Config.Users.draco.email
        }
        withSignIn(Config.Users.snape) { listPage =>
          listPage.readUserEmail() shouldEqual Config.Users.snape.email
        }
        withSignIn(Config.Users.draco) { listPage =>
          listPage.readUserEmail() shouldEqual Config.Users.draco.email
        }
      }
    }
```
Just need to add `with Matchers` to the class declaration to get the `should` matchers (improvement over `assert`).